### PR TITLE
edit number two

### PR DESCRIPTION
**Pull request by jollyyas**
### What problem does this solve?

Fixes #244 

Context: perviously margins were unaligned 

### How does this solve it?
edits for different devices

### QA
NA

### CODE REVIEW

**Comment by jeff-rey-wang** [comment]
Do not merge, will continue to add some changes to this PR before merging

**Comment by jeff-rey-wang** [comment]
Complete list of changes:
- Updated style of buttons on profile page
- Widened horizontal margins across the entire site
- Added image on profile page 
- Fixed upload page background to fit content (no more partially green background)

Note: Will not be creating a new display component on the upload page (unless anyone thinks it is absolutely necessary)

**Review by jsrebrnjak — APPROVED** [review]

**Pull request merged**
